lib/constants.py

```diff
@@ -19,7 +19,7 @@
 # Region api is created in.  Later versions of boto3 should allow us to
 # extract this from the session variable.  Hard coding for now.
 REGION = 'us-east-1'
-INCOMING_SUBNET = "52.3.13.189/32"  # microns-bastion elastic IP
+INCOMING_SUBNET = os.environ['_BASTION_ALLOW_IP'] + "/32"  # microns-bastion elastic IP
```

Comment:
1. You never added `_BASTION_ALLOW_IP` to `set_vars.sh` for developers
2. I would do `os.environ.get('_BASTION_ALLOW_IP', '52.3.13.189') + '/32'` so that existing developers don't have to change anything